lightning.install

```diff
@@ -9,6 +9,34 @@ use Drupal\Core\Entity\Entity\EntityViewDisplay;
 use Drupal\field\Entity\FieldConfig;
 use Drupal\lightning_core\ConfigHelper as Config;
 
+/**
+ * Alerts users of Libraries API dependencies.
+ *
+ * A future version of Lightning will remove Libraries API, so
+ * users must add that dependency to composer.json manually.
+ */
+function lightning_requirements() {
+    $requirements = [];
+    $libraries_dependencies = [];
+    $enabled_modules = \Drupal::moduleHandler()->getModuleList();
+    
+    foreach ($enabled_modules as $module => $data) {
+        $info = \Drupal::service('extension.list.module')->getExtensionInfo($module);
+        if (in_array('drupal:libraries', $info['dependencies'])) {
+            $libraries_dependencies[$module] = $info['name'];
+        }
+    }
+
+    if ($enabled_modules) {
+        $requirements['lightning_libraries'] = [
+            'title' => t('Libraries API dependencies'),
+            'value' => t('These modules require Libraries API: %module_list. Lightning will soon remove Libraries API as a dependency, so you must add that dependency to your composer.json manually.', ['%module_list' => implode(', ', $libraries_dependencies)]),
```

Comment:
There's a nice helper function in Lightning Core that we can use to improve the formatting of the comma-separated list. See `\Drupal\lightning_core\Element::oxford()`. :)